vaadin-tabs-flow-parent/vaadin-tabs-flow/src/test/java/com/vaadin/flow/component/tabs/tests/TabsTest.java

```diff
@@ -22,11 +22,12 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+import com.vaadin.flow.component.Component;
 import com.vaadin.flow.component.tabs.Tab;
 import com.vaadin.flow.component.tabs.Tabs;
 
 import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Is this change intended?

Reply:
Yes, org.junit.Assert.assertThat is deprecated, so I replaced it with org.hamcrest.MatcherAssert.assertThat to get rid of the warnings

Reply:
Oh, alright.